src/components/Navigation/Navigation.js

```diff
@@ -35,9 +39,22 @@ const NavigationStyled = styled.div`
   background: #bcbbbc;
 `
 
+const HomeIconStyled = styled(IconHome)`
+  z-index: 100;
+`
+
+const OverviewIconStyled = styled(IconMonthlyOverview)`
+  z-index: 100;
+`
+
 const AddIcon = styled(IconAdd)`
   z-index: 100;
   position: relative;
   bottom: 30px;
   filter: drop-shadow(1px 1px 5px rgba(0, 0, 0, 0.3));
+  border-radius: 50%;
+  background-color: ${(props) => (props.createPath ? '#ffb9b9' : '#fff')};
```

Comment:
```suggestion
  background-color: ${(props) => (props.createPath ? '#ffb9b9' : '#ffffff')};
```
In your `src/components/Navigation/Navigation.js`you are using #ffffff. I guess you shouldn't swith to #fff even if it is the same color ;-) (But also here: maybe you can add css variables ;-))